src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -54,6 +54,11 @@ def exists(self, query):
         query = self._apply_kwargs(query)
         return JMESPathCheckExists(query)
 
+    def not_exists(self, query):
+        from azure.cli.testsdk.checkers import JMESPathCheckNotExists
+        query = self._apply_kwargs(query)
+        return JMESPathCheckNotExists(query)
+
```

Comment:
LGTM, just curious that why this common-used function was not added at the first time.